BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -101,14 +101,12 @@
 "EditItem" = "Edit item";
 "EnableAutomaticSyncing" = "Allow automatic syncing";
 "EnterEmailForHint" = "Enter your account email address to receive your master password hint.";
-"ExntesionReenable" = "Reactivate app extension";
+"ExtensionReenable" = "Reactivate app extension";
```

Comment:
⛏️ Given the key is being modified, could we change the it to `ReactivateAppExtension`?

Reply:
sure!  makes sense.  I just saw the spelling error and figured id fix it but this makes more sense.  Thank you!